hail/python/setup.py

```diff
@@ -27,7 +27,7 @@
             dependencies.append(pkg)
 
 setup(
-    name="cpg-hail",
+    name="hail",
```

Comment:
I think this one should stay `cpg-hail` in case we want to release our own version - we should also change the fields so that our cpg-hail has the correct updated links.

Reply:
I'll merge this now, to get some dependent PRs in, but probably worth revisiting this point in a follow-up PR?